schemas/dab.draft.schema.json

```diff
@@ -175,6 +175,23 @@
             "enabled": {
               "type": "boolean",
               "description": "Allow enabling/disabling GraphQL requests for all entities."
+            },
+            "nested-mutations": {
+              "type": "object",
+              "description": "Feature flags for nested mutation operations",
+              "properties": {
+                "inserts":{
+                  "type": "object",
+                  "description": "Options for nested insert operations",
```

Comment:
nit: nested create, please make the change from insert to create wherever applicable.

Reply:
I've updated all the relevant places